doc/TUTORIAL.md

```diff
@@ -121,22 +121,21 @@ cache(function(data, match, sendBadge, request) {
   var url = 'https://registry.hub.docker.com/v2/repositories/' + path;           // (4)
   var badgeData = getBadgeData('docker build', data);                            // (5)
   request(url, function(err, res, buffer) {                                      // (6)
-    if (err != null) {                                                           // (7)
-      badgeData.text[1] = 'inaccessible';                                        // (7)
+    if (checkErrorResponse(badgeData, err, res, 'repo not found')) {             // (7)
       sendBadge(format, badgeData);                                              // (7)
       return;                                                                    // (7)
     }
     try {
-      var data = JSON.parse(buffer);                                             // (8)
-      var is_automated = data.is_automated;                                      // (8)
+      var parsedData = JSON.parse(buffer);                                       // (8)
+      var is_automated = parsedData.is_automated;                                // (8)
       if (is_automated) {
         badgeData.text[1] = 'automated';                                         // (9)
         badgeData.colorscheme = 'blue';                                          // (9)
       } else {
         badgeData.text[1] = 'manual';                                            // (9)
         badgeData.colorscheme = 'yellow';                                        // (9)
       }
-      badgeData.colorB = '#008bb8';                                              // (9)
+      badgeData.colorB = data.colorB || '#008bb8';                               // (9)
```

Comment:
Should `makeColorB` be used here?

Reply:
Probably a good idea, didn't even realise this was an option, it's currently only used in 5 places.

Reply:
This is actually a bug in my opinion. I wanted to update an example with code from `server.js` and fix this bug in a separate PR. 
`colorB` query param can be used to override badge value color. 
This is how it works (000000 is black, it works out of the box):
https://img.shields.io/github/downloads/atom/atom/total.svg?colorB=000000
But it does not work for docker automated badge:
https://img.shields.io/docker/automated/jrottenberg/ffmpeg.svg?colorB=000000
https://img.shields.io/docker/automated/_/ubuntu.svg?colorB=000000
Actually this line:
```js
badgeData.colorB = data.colorB || '#008bb8';
```
overrides colorscheme defined few lines above. 
Overriding is implemented here:
https://github.com/badges/shields/blob/3518b642e7a37f582692cfad0761859da0bf2b2c/lib/make-badge.js#L128
And docker badge always has color `#008bb8`
https://img.shields.io/docker/automated/jrottenberg/ffmpeg.svg
https://img.shields.io/docker/automated/_/ubuntu.svg

Reply:
Hmmm. Okay, let's deal with this separately.